test/helper.cpp

```diff
@@ -46,8 +46,9 @@ int main()
     TEST(!isEqualF(1.f, NAN));
     TEST(!isEqualF(INFINITY, 1.f));
     TEST(!isEqualF(1.f, INFINITY));
-    TEST(!isEqualF(NAN, NAN));
-    TEST(!isEqualF(INFINITY, INFINITY));
+    TEST(isEqualF(NAN, NAN));
+    TEST(isEqualF(NAN, -NAN));
+    TEST(isEqualF(INFINITY, INFINITY));
```

Comment:
What about 
`TEST(!isEqualF(INFINITY, -INFINITY));`

Reply:
Sure, wait a mom.

Reply:
How should I check that if I can't use == with the float? -INFINITY !< 0

Reply:
I found an efficient way 🎉

Reply:
https://github.com/PX4/Matrix/pull/93/commits/e9157f71a350c0df3ed31c6d00935209508e947c